partiql-lang/src/main/kotlin/org/partiql/lang/graph/ExternalGraphReader.kt

```diff
@@ -1,14 +1,25 @@
 package org.partiql.lang.graph
```

Comment:
I believe all of the internal methods in this class could actually be private.

Reply:
Good point -- except for the `read()` methods. 

The fact that those are `internal` is because of `SimpleGraph` in their return type, which I had designated internal so far.  I am not sure yet where this will end up eventually.  These read methods ought to be public, but I did not yet figure out how, to make that possible, to change their return type to a public type (`Graph` would work) while still being able to do tests in `ExternalGraphReaderTests` (which need methods from `SimpleGraph` that `Graph` does not have). I hope there is some king of a test/package wizardry that can reconcile this tension.

Reply:
As you've said, let's do private for everything except that which is exposed for tests being internal. And correct, public methods cannot return internal/private types.

Reply:
Done.